src/control/server/storage/bdev_test.go

```diff
@@ -0,0 +1,70 @@
+//
+// (C) Copyright 2022 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package storage
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/daos-stack/daos/src/control/common"
+)
+
+func Test_NvmeDevState(t *testing.T) {
+	// verify "NEW" state
+	state := NvmeStatePlugged
+	common.AssertTrue(t, state.IsNew(), "expected state to indicate new")
+	common.AssertFalse(t, state.IsNormal(), "expected state to not indicate normal")
+	common.AssertFalse(t, state.IsFaulty(), "expected state to not indicate faulty")
+	common.AssertTrue(t, NvmeDevStateFromString(state.StatusString()) == state,
+		fmt.Sprintf("expected string %s to yield state %s", state.StatusString(),
+			NvmeDevStateFromString(state.StatusString()).StatusString()))
+
+	// verify "NORMAL" state
+	state = NvmeStatePlugged | NvmeStateInUse
+	common.AssertFalse(t, state.IsNew(), "expected state to not indicate new")
+	common.AssertTrue(t, state.IsNormal(), "expected state to indicate normal")
+	common.AssertFalse(t, state.IsFaulty(), "expected state to not indicate faulty")
+	common.AssertTrue(t, NvmeDevStateFromString(state.StatusString()) == state,
```

Comment:
Could you use AssertEqual for these checks?

Reply:
will address if I have to re-push